package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
You are missing a start script in here, and the script for your test isn't going to allow tests to run. If you add this, that will fix the problems:
```
"start": "node index.js",
"test": "jest --verbose --coverage",
"test-watch": "jest --watchAll --verbose --coverage"
```